plugins.txt

```diff
@@ -1,4 +1,4 @@
-blueocean:1.8.2
-workflow-support:incrementals;org.jenkins-ci.plugins.workflow;2.21-rc591.43d37d4d080a
+blueocean:1.9.0
+workflow-support:experimental
```

Comment:
LGTM, so I suppose this will point to the `temporary-experimental-java11...` one?

Reply:
yes. Works on my machine